dev-requirements.txt

```diff
@@ -19,3 +19,4 @@ asgiref==3.7.2
 psutil==5.9.6
 GitPython==3.1.41
 flaky==3.7.0
+ruff==0.3.4
```

Comment:
I think you can drop the first 4 lines from this file

Reply:
Probably more lines too, like astroid and other transitive deps of these linters. Will take a look

Reply:
Addressed in new PR